### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by chepkok3**
- [ ] Set up a new project with webpack.
- [ ] Created an index.html file.
- [ ] Created an index.js file and set an array of simple to-do tasks. Each task object contains three keys:
- description [string].
- completed [bool].
- index: [number].
- [ ] Wrote a function to iterate over the tasks array and populate an HTML list item element for each task.
- [ ] Created a style.css and set rules for the To Do List. The CSS is loaded by Webpack Style/CSS Loader.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    <script defer src="main.js"></script>
+  </head>
+  <body>
+    <main class="main-content">
+      <section class="to-do">
+        <div class="todo-container">
+          <ul class="task-list"></ul>
+          <div class="completed">
+            <button class="delete-tasks">Clear all completed</button>
+          </div>
```
Comment (ShadyShawkat) [inline src/index.html:28]: - [ ] Kindly make sure that the "Clear all completed" button matches the original design.

| Your design | Original design |
| --- | --- |
| image | image |

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      <div class="list">
+        <input type="checkbox">
+        <label for="description" class="task-description">${task.description}</label>
+        <iconify-icon icon="ph:dots-three-outline-vertical-fill" class="dots"></iconify-icon>
+    <hr> 
+      </div
+      `;
+
+  taskList.appendChild(toDoTask);
+});
```
Comment (ShadyShawkat) [inline src/index.js]: - [ ] Please make sure that the list is sorted according to their individual index value, as per [this](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=The%20list%20should%20appear%20in%20order%20of%20the%20index%20values%20for%20each%20task.) requirement.

- [ ] [OPTIONAL] Kindly wrap the functionality of populating the DOM with the to-do list in a function to make it easier to call in the future.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @chepkok3, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @chepkok3 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- GitHub flow was used correctly 🚀  
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**